tests/test_stdlib.py

```diff
@@ -495,9 +495,9 @@ def test_foreign_delegate(self, configure_for_pf, capsys):
 
         assert ("", "foo [in test_foreign_delegate]\n") == capsys.readouterr()
 
-    def test_clears_args(self, capsys, configure_for_pf):
+    def test_clears_args(self, configure_for_pf, capsys):
```

Comment:
I changed this so it's according to style in whole class but I'll remove the change if that's a problem.